uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveRequestStatusMatrixTest.php

```diff
@@ -73,28 +73,101 @@ public function testCanTransitionToForLeaveApproverReturnsFalseForAllNonPossible
   }
 
   public function testCanTransitionToForAdminForReturnsTrueAllPossibleTransitionStatuses() {
+    $adminID = 5;
```

Comment:
Well it won't because all that is isn't interfering with any other test. All it does is log in the contact ID supplied in session and we don't need an actual contact fabricated for that